### DISCUSSION TITLE
0.0.11

### DISCUSSION THREAD

**Discussion by swetoast** [swetoast/steamlink-launcher#39] [General]
Bugfixes should boot now with Kodi 19 

<hr /><em>This discussion was created from the release <a href='https://github.com/swetoast/steamlink-launcher/releases/tag/0.0.11'>0.0.11</a>.</em>

**Comment by AStefan93** [comment]
I've installed it on my raspberry pi 4 with osmc. I have the same issue as before it seems. Steamlink starts, but there's no video output.

**Comment by swetoast** [comment]
ask for support on the steam forums cause the launcher launched steamlink
